src/styles/components/_component-card.scss

```diff
@@ -48,9 +48,7 @@
       z-index: 1;
       font-size: 1.125rem; // 18px/16
       line-height: 4.6667rem; // 75px
-      letter-spacing: 0.02em;
-      border-top: 0.25rem solid var(--foreground-color);
-      background-color: var(--background-color);
+      background-color: var(--foreground-color);
       position: relative;
       text-align: center;
       display: block;
```

Comment:
I think we still need these properties that got deleted. I also had to add a quick dark mode query to make sure the headers backgrounds weren't black! 😆 Otherwise, on dark mode, the header background was white ✅, but the light-mode header background was black and you couldn't read the text! 😩

This is what I did starting at line 51:
```
    background-color: var(--background-color);
      position: relative;
      text-align: center;
      display: block;
      letter-spacing: 0.02em;
      border-top: 0.25rem solid var(--foreground-color);

      @media (prefers-color-scheme: dark) {
        border-top: 0.25rem solid var(--background-color);
        background-color: var(--foreground-color);
      }
```

Reply:
Oh yeah! This is exactly what my other comment was about! We want the letter-spacing and border, etc to all be here automatically at small screens, so we don't need them in the media query later in the file. They're already here!

Reply:
Yup, it looks good now--thank you! :)

![image](https://user-images.githubusercontent.com/49820744/200974987-9fa7abef-553b-45df-a8e5-c767aa9eecb7.png)